docs/data-sources/users.md

```diff
@@ -0,0 +1,80 @@
+---
+subcategory: "Security"
+---
+
+# databricks_users Data Source
+
+-> If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../guides/troubleshooting.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _default auth: cannot configure default credentials_ errors.
```

Comment:
Can we add a note that it works with both workspace & account-level providers?

Reply:
Done!